hack/build/docker/builder/Dockerfile

```diff
@@ -23,7 +23,8 @@ RUN \
     go install ./... ) && \
     ( go get -d github.com/mattn/goveralls && \
     cd $GOPATH/src/github.com/mattn/goveralls && \
-    go install ./... )
+    go install ./... ) && \
+    ( go get -u golang.org/x/lint/golint )
```

Comment:
Do we need a 
cd $GOPATH/src/golang.org/x/lint/golint && \
go install ./...
?

Reply:
Yeah, I intentionally did not use the "-d" flag here for that reason.  I wasn't sure if there was a reason for the pattern here of using "got get -d" and then manually installing.  "go get -d" is special in that it only downloads and skips the install.  Things like "go get -u" however will download (including update) and install for you so you don't need the extra steps.

I figured there might be a reason for doing it the way it is here but for the case of golint it just seemed like an unnecessary extra step.